### ISSUE TITLE
API - https://www.opencve.io/api/cve

### ISSUE THREAD

**Issue by yuri-spm** [opencve/opencve#362]
Can I use your API to integrate into other systems?

I'm trying to access via Postman and it's asking for username and password. I put in my email and my registered password and it says I'm not authorized.

Where can I get this authorization?

**Comment by ncrocfer** [comment]
Indeed OpenCVE v1 API uses basic authentication, you can use your username & password: https://docs.opencve.io/api/#authentication

**Comment by yuri-spm** [comment]
So I signed up on the website and filled out my profile. I'm using my username and password that I registered there. Do I need to obtain a username and password from somewhere else? If so, could you send me the link?

**Comment by ncrocfer** [comment]
It's the same as you registered. Maybe if you give your request context (payload, header...) it will help us to understand your issue.

**Comment by yuri-spm** [comment]
I'm trying to use the OpenCVE API in Postman, for personal study, but when I try to log in, it gives me this error:

![image](https://github.com/opencve/opencve/assets/63649751/709599ca-3897-4911-9148-f415c4bfc222)

**Comment by ncrocfer** [comment]
I think you have an issue in your Postman configuration, as the API of OpenCVE is well working:

```
$ curl -u john https://www.opencve.io/api/cve/CVE-2024-3400
Enter host password for user 'john':
{"id": "CVE-2024-3400", "summary": "A command injection vulnerability in the GlobalProtect feature of Palo Alto Networks PAN-OS software for specific PAN-OS versions and distinct feature configurations may enable an unauthenticated attacker to execute arbitrary code with root privileges on the firewall.\n\nFixes for PAN-OS 10.2, PAN-OS 11.0, and PAN-OS 11.1 are in development and are expected to be released by April 14, 2024. Cloud NGFW, Panorama appliances, and Prisma Access are not impacted by this vulnerability. All other versions of PAN-OS are also not impacted.", "created_at": "2024-04-12T08:15:06Z", "updated_at": "2024-04-12T08:15:06Z", "cvss": {"v2": null, "v3": 9.8}, "vendors": {}, "cwes": ["CWE-77"], "raw_nvd_data": {"id": "CVE-2024-3400", "metrics": {"cvssMetricV31": [{"type": "Secondary", "source": "psirt@paloaltonetworks.com", "cvssData": {"scope": "UNCHANGED", "version": "3.1", "baseScore": 9.8, "attackVector": "NETWORK", "baseSeverity": "CRITICAL", "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H", "integrityImpact": "HIGH", "userInteraction": "NONE", "attackComplexity": "LOW", "availabilityImpact": "HIGH", "privilegesRequired": "NONE", "confidentialityImpact": "HIGH"}, "impactScore": 5.9, "exploitabilityScore": 3.9}]}, "published": "2024-04-12T08:15:06.230", "references": [{"url": "https://security.paloaltonetworks.com/CVE-2024-3400", "source": "psirt@paloaltonetworks.com"}], "vulnStatus": "Received", "weaknesses": [{"type": "Secondary", "source": "psirt@paloaltonetworks.com", "description": [{"lang": "en", "value": "CWE-77"}]}], "descriptions": [{"lang": "en", "value": "A command injection vulnerability in the GlobalProtect feature of Palo Alto Networks PAN-OS software for specific PAN-OS versions and distinct feature configurations may enable an unauthenticated attacker to execute arbitrary code with root privileges on the firewall.\n\nFixes for PAN-OS 10.2, PAN-OS 11.0, and PAN-OS 11.1 are in development and are expected to be released by April 14, 2024. Cloud NGFW, Panorama appliances, and Prisma Access are not impacted by this vulnerability. All other versions of PAN-OS are also not impacted."}], "lastModified": "2024-04-12T08:15:06.230", "sourceIdentifier": "psirt@paloaltonetworks.com"}}
```

Please change your client or check your Postman configuration. I will close this ticket as the web credentials is the same as the API ones. It's not a bug in OpenCVE but in your own usage :)
